jenkins/debian/Jenkinsfile_basic

```diff
@@ -0,0 +1,17 @@
+library identifier: 'jenkins-libs@master', retriever: modernSCM(
+  [$class: 'GitSCMSource',
+   remote: 'https://github.com/ana/kernelci-jenkins-libs'])
```

Comment:
Should these be in a main kernelci repo, instead of your personal one?

Reply:
Yes, these should be in a kernelci repo. They could be in be the same with the Jenkinsfiles if we don't plan to use them from different repositories or branches. 
However, having them in a separate repository is a good idea for several reasons:
- shared libraries must follow this directory structure https://jenkins.io/doc/book/pipeline/shared-libraries/#directory-structure
- we can create branches in the same repository when we need to make big changes or need to test a specific change.
- You can add the shared library directly to the jenkins configuration and use the in the pipeline syntax checker directly when playing/testing with the pipelines.